### PR TITLE
factor out btagDiscriminators definition for easier modification

### PR DESCRIPTION
**Pull request by slava77**
this is inspired by the needs in #25370

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

File: PhysicsTools/PatAlgos/python/slimming/applyDeepBtagging_cff.py [hunk 1]
```diff
@@ -59,20 +74,7 @@ def applyDeepBtagging( process, postfix="" ) :
...
-          'pfDeepDoubleCvBJetTags:probHbb',
-          'pfDeepDoubleCvBJetTags:probHcc',
-          'pfMassIndependentDeepDoubleBvLJetTags:probQCD',
-          'pfMassIndependentDeepDoubleBvLJetTags:probHbb',
-          'pfMassIndependentDeepDoubleCvLJetTags:probQCD',
-          'pfMassIndependentDeepDoubleCvLJetTags:probHcc',
-          'pfMassIndependentDeepDoubleCvBJetTags:probHbb',
-          'pfMassIndependentDeepDoubleCvBJetTags:probHcc',
-       ] + pfDeepBoostedJetTagsAll,
+       btagDiscriminators = _btagDiscriminators.names,
```
Comment (makortel) [inline PhysicsTools/PatAlgos/python/slimming/applyDeepBtagging_cff.py]: I'd suggest
```suggestion
       btagDiscriminators = _btagDiscriminators.names.value(),
```
Then, if anyone would modify `_btagDiscriminators.names` later in this function, those changes would **not** propagate here (as would happen now). Unless that would be the intention, of course.

And as an added value, then the right-hand-side would be without the `cms.` type so the "typo checking" would still be done.

**Comment by slava77** [comment]
@cmsbuild please test

**Comment by slava77** [comment]
+1

for 
- this is a somewhat technical refactoring in a python configuration method
- jenkins tests pass and comparisons with the baseline show no differences, as expected

**Comment by fabiocos** [comment]
+1

**Comment by fabiocos** [comment]
merge

**Pull request merged**